<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="Nikhil" failures="1" tests="3" name="Test" time="18.1" errors="0" timestamp="2023-05-31T18:49:34 IST">
  <testcase classname="testcase.StandAloneTest" name="display" time="4.286"/>
  <testcase classname="testcase.StandAloneTest" name="display" time="4.362"/>
  <testcase classname="testcase.WrongCredentials" name="loginWithIncorrectCredentials" time="0.414">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at testcase.WrongCredentials.loginWithIncorrectCredentials(WrongCredentials.java:16)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- loginWithIncorrectCredentials -->
</testsuite> <!-- Test -->
